### PR TITLE
[WPMangaReader] Theme Update - Fixes Pagination 

### LINKED ISSUE #6023: [Bug] Hikari Scan (pt) - Manga repetition
## Device information
* Tachiyomi version: 0.10.9
* Android version: 7

## Source information
* Source name: Hikari Scan
* Source extension version: 1.2.2

## Steps to reproduce
1. Click on the extension or latest

## Issue/Request
There is a repetition of the five manga, both the latest and the source. The same five manga are repeated over and over.

## Other details
Thanks for updating the extension :)

### PR DESCRIPTION
**Pull request by happywillow0**
- Fixes page URLs / Closes #6023
- Removed unnecessary Ngomik override
- Incremented version code

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt [hunk 1]
```diff
@@ -9,12 +9,12 @@ class WPMangaReaderGenerator : ThemeSourceGenerator {
...
     override val themeClass = "WPMangaReader"
 
-    override val baseVersionCode: Int = 1
+    override val baseVersionCode: Int = 2
 
     override val sources = listOf(
             SingleLang("Hikari Scan", "https://hikariscan.com.br", "pt-BR", overrideVersionCode = 1),
             SingleLang("KomikMama", "https://komikmama.net", "id"),
-            SingleLang("MangaKita", "https://mangakita.net", "id"),
+            SingleLang("MangaKita", "https://mangakita.net", "id"), //Might be WPMangaStream
```
Comment (Riztard) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt:17]: their website use css from mangareader if you inspect their site

Comment (happywillow0) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt:17]: Cool, the overall style looked like Mangareader but the details section didn't match the rest. It'll be on the todo list to override it to pull in the proper data in for someone in the future then.

**Comment by CarlosEsco** [comment]
Is this good to merge?

**Comment by happywillow0** [comment]
Yes. Everything looks good to me.

**Pull request merged**